package.json

```diff
@@ -9,14 +9,21 @@
     "@anthropic-ai/tokenizer": "^0.0.4",
     "@inquirer/prompts": "^4.3.0",
     "cross-spawn": "^7.0.3",
+    "directory-tree": "^3.5.1",
+    "mime-types": "^2.1.35",
+    "next": "^14.1.4",
```

Comment:
Haha it happened again

Reply:
What happened again? Do I need to remove `next`?